lib/herodot/output.rb

```diff
@@ -48,8 +48,12 @@ def colorize(project)
       Rainbow(project).color(COLORS[project.chars.map(&:ord).reduce(:+) % COLORS.size])
     end
 
+    def times_by_project_and_branch(times)
+      times.sort_by { |log| log[:branch] }.sort_by { |log| log[:project] }
```

Comment:
Eh, maybe my Ruby is a bit rusty: Can you use `Enumerable#sort_by` twice?
Is the second run guaranteed to preserve the ordering within groups of entries for the same project?

I was expecting something like the following:

```ruby
times.sort_by { |log| [log[:project], log[:branch]] }
```